Error running cdk deploy on capitalize-string

## :bug: Bug Report

### What is the problem?
Running the "capitalize-string" causes an error in "cdk deploy"
https://github.com/aws-samples/aws-cdk-examples/tree/master/csharp/capitalize-string#this-project

Ref: MSBUILD : error MSB1008: Only one project can be specified.

This seems to be related to cdk.json (https://github.com/aws-samples/aws-cdk-examples/blob/master/csharp/capitalize-string/cdk.json) where 2 builds are specified

### Reproduction Steps
Clone the repository and follow the readme here: 
https://github.com/aws-samples/aws-cdk-examples/tree/master/csharp/capitalize-string#build--deploy

### Verbose Log

   cdk deploy
   (node:24072) ExperimentalWarning: The fs.promises API is experimental   `
   Microsoft (R) Build Engine version 16.6.0+5ff7b0c9e for .NET Core
   Copyright (C) Microsoft Corporation. All rights reserved.
   
   MSBUILD : error MSB1008: Only one project can be specified.
   Switch: dotnet
   
   For switch syntax, type "MSBuild -help"
   Subprocess exited with error 1


### Environment

  - **CDK CLI Version:** 1.41.0 (build 9e071d2)
  - **Example:** capitalize-string
  - **Example Version:** <!-- Version of the module in question -->
  - **OS:** Windows 10
  - **Language:** csharp

--- a/csharp/capitalize-string/src/CapitalizeString/CapitalizeString.csproj
+++ b/csharp/capitalize-string/src/CapitalizeString/CapitalizeString.csproj
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <!-- Roll forward to future major versions of the netcoreapp as needed -->
     <RollForward>Major</RollForward>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.CDK.Lib" Version="2.0.0" />
+    <PackageReference Include="Amazon.CDK.Lib" Version="2.173.4" />
     <PackageReference Include="Constructs" Version="10.0.0" />
   </ItemGroup>
 

--- a/csharp/capitalize-string/src/CapitalizeString/CapitalizeStringStack.cs
+++ b/csharp/capitalize-string/src/CapitalizeString/CapitalizeStringStack.cs
@@ -10,7 +10,7 @@ namespace CapitalizeString
         {
             Function fn = new Function(this, "capitalizestring", new FunctionProps
             {
-                Runtime = Runtime.DOTNET_CORE_3_1,
+                Runtime = Runtime.DOTNET_8,
                 Code = Code.FromAsset("./CapitalizeStringHandler/src/CapitalizeStringHandler/bin/Release/net6.0/publish"),
                 Handler = "CapitalizeStringHandler::CapitalizeStringHandler.Function::FunctionHandler"
             });

